### ISSUE TITLE
После BK/FDB базы данных не сохраняется значение в таблице "AC_ACCOUNT" в поле "Поле для отображения на английском"

### ISSUE THREAD

**Issue by gsbelarus** [GoldenSoftwareLtd/gedemin#2602]
Originally reported on Google Code with ID 2602

```

    Обычно здесь удобно прописать ALIAS. Тогда. при построении журнал-ордера, оборотной
ведомости в поле "Счет" будет отображаться цифровое обозначение счета, а не его текстовая
интерпретация.
    Не совсем удобно после BK/FDB базы данных заново прописывать значение в таблице
AC_ACCOUNT.

```

Reported by `NikolayUkleyko` on 2011-10-18 11:26:26

**Comment by gsbelarus** [comment]
```
BK/FDB ничего не делает с данными в БД. так что тут дело в чем-то другом, но разобраться
надо.
```

Reported by `gs1994` on 2011-10-18 11:28:38
- **Status changed**: `Started`
- **Labels added**: Type-Defect, Priority-Medium, GedeminEXE

**Comment by gsbelarus** [comment]
```
Значение этого поля часто настройками затирается, br/restore тут не причем
```

Reported by `Alexander.GoldenSoft` on 2011-10-18 11:49:04

**Comment by gsbelarus** [comment]
```
Надо посмотреть в какие настройки входит и с какими флагами. Почему затирается.
```

Reported by `gs1994` on 2011-10-18 12:13:20

**Comment by gsbelarus** [comment]
```
затирается когда с другой базы делается настройка, когда процедура с таблицой AC_ACCOUNT
переносится... то же происходит и с другими таблицами (GD_GOOD часто добавляют ALIAS,
gd_document DOCUMENTDATE и т.д.)
```

Reported by `mikleshoihet` on 2011-10-18 12:50:57

**Comment by gsbelarus** [comment]
```
Может обрабатывать CustomModify и если загрузка из потока и поле LISTFIELD не пустое,
то не обновлять его.
```

Reported by `Alexander.GoldenSoft` on 2011-10-18 14:58:03

**Comment by gsbelarus** [comment]
```
Можно и так. Только что бы следующий, кто с этим столкнется не сошел с ума. Почему в
настройке есть значение, а на базу не грузится.
```

Reported by `gs1994` on 2011-10-18 14:59:54

**Comment by gsbelarus** [comment]
```
Тут палка о двух концах...что лучше, если сломается что-то настроенное, или что-то настроенное
не обновится на новое значение.
```

Reported by `Alexander.GoldenSoft` on 2011-10-18 15:04:18

**Comment by gsbelarus** [comment]
```
А добавлять новые поля к тем, что уже были, можно?
```

Reported by `alexandra.gsoftware` on 2011-10-18 15:09:09

**Comment by gsbelarus** [comment]
```
кстати да, можно новое значение добавлять к старому.
```

Reported by `Alexander.GoldenSoft` on 2011-10-18 15:15:19

**Comment by gsbelarus** [comment]
Reported by `NikolayUkleyko` on 2013-05-21 14:36:55
- **Status changed**: `Duplicate`
- **Merged into**: #3019
